projects/plugins/jetpack/_inc/client/search/dashboard/index.jsx

```diff
@@ -34,14 +37,24 @@ const useComponentWillMount = func => {
  */
 function SearchDashboard( props ) {
 	// NOTE: API root and nonce must be set before any components are mounted!
-	const { apiRootUrl, apiNonce, setInitialState: setSearchDashboardInitialState } = props;
+	const {
+		apiRootUrl,
+		apiNonce,
+		setInitialState: setSearchDashboardInitialState,
+		siteConnectionStatus,
```

Comment:
What's the value of `siteConnectionStatus`? Is it a string enum? Could we rename this variable so that it's more obvious?

Reply:
removed as it's unnecessary